modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -1582,6 +1582,7 @@ and=And
 and-contains=AND Contains
 and-operator=And Operator
 and-or-operator=And/Or Operator
+and-replaced-by-objects.-ddl-will-be-removed-in-2024=and replaced by Objects. DDL will be removed in 2024.
```

Comment:
I see that this language key represent the "middle" of a sentence. I understand the reason you did, it is because the design set this pattern to you. I will open a thread with you and Julia because I don't think Brian will accept this language key. 
There is no language key in the portal representing half/part of a sentence.

Reply:
As discussed on Slack,
![image (1)](https://github.com/liferay-forms/liferay-portal/assets/15161924/fdb7fad0-ad90-423c-860d-0c9acb13c0bf)

Reply:
Thanks, @marcelabc! I'm working on this changes